options/locale/locale_cs-CZ.ini

```diff
@@ -908,7 +908,6 @@ issues.dependency.add_error_dep_issue_not_exist=Související úkol neexistuje.
 issues.dependency.add_error_dep_not_exist=Závislost neexistuje.
```

Comment:
You MUST not change the translation files directly but go to https://crowdin.com/project/gitea

Reply:
sorry...I will revert these...in this case I was removing an error message that is no longer used with this change. if there's anything special to this case let me know or I'll just read on crowdin to see what I need to do thanks